index.html

```diff
@@ -33,6 +33,9 @@ <h1 id="Display Name"></h1>
         <img src="./pages/assets/hearthstone.jpg" alt="Broken Link">
       </div>
       <form action="./pages/game.html" id="Player">
+        <!-- todo: forgot to add space between 'name' and 'id' properties -->
+        <!-- todo: prefer a 'label' element over the 'placeholder' property
+              https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Labels_and_placeholders -->
         <input type="text" name="playerName"id="name" placeholder="Enter player name here!"><br>
         <input type="submit" value="PLAY"   >
```

Comment:
Can be read about here: [Labels and Placeholders](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Labels_and_placeholders)

![2019-06-27_01-19-49](https://user-images.githubusercontent.com/30473157/60241769-a8aebf00-9879-11e9-8c2f-bea5db7abbfa.png)

Your intended use is to provide a nifty label within the box. Just keep in mind the issues this might cause in other projects. Obviously, it's not going to be a problem here.